schema/bom-1.5.schema.json

```diff
@@ -1492,6 +1492,7 @@
         "CVSSv2",
         "CVSSv3",
         "CVSSv31",
+        "CVSSv4",
```

Comment:
The `description` was not updated for JSON,
whitest the `documentation` was updated for XML.

Reply:
Good catch. The JSON docs have been updated to include CVSSv4